doc/grammars/type.abnf

```diff
@@ -35,7 +35,13 @@ GenericTypeArgument
     / TokenWildcard
 
 Callable
-	= TokenParenthesesOpen [CallableParameters] TokenParenthesesClose TokenColon CallableReturnType
+	= [CallableTemplate] TokenParenthesesOpen [CallableParameters] TokenParenthesesClose TokenColon CallableReturnType
```

Comment:
@ondrejmirtes can I ask you for a hint how to fix the fuzzer tests? The grammar seems correct to me, and the manualy tests are passing as expected...